__tests__/app.test.js

```diff
@@ -1,40 +1,32 @@
 const request = require('supertest');
-const data = require('../db/data/test-data/index.js');
+const data = require('../db/data/development-data/index.js');
```

Comment:
You should not be testing on dev data. The seed file will default any votes to 0 that aren't present on an article.